grc/core/params/param.py

```diff
@@ -308,6 +308,8 @@ def to_code(self):
                 self.evaluate()
             return '[' + value + ']' if self._lisitify_flag else value
         else:
+            if self.dtype in ('int', 'real') and ('+' in value or '-' in value):
```

Comment:
what about other arithmetic operators?

Reply:
While trying to justify my choice I came across, that we need parentheses in some cases for * as

a = b/c*d != a=b/(c*d)

So it's better to add () for +,-,*,/